routes/user.js

```diff
@@ -0,0 +1,96 @@
+"use strict";
+
+const AppConstants = require("../app-constants");
+
+const crypto = require("crypto");
+const express = require("express");
+const router = express.Router();
+
+const EmailUtils = require("../email-utils");
+const gEmails = require("../subscribers");
+
+// We send verification emails to addresses that want to subscribe.
+// These addresses are temporarily stored here, mapped to the unique
+// string token that is used for verification.
+var gUnverifiedEmails = new Map();
+
+router.post("/add", function(req, res) {
+  // TODO: use a hash of the email address instead of a random string.
+  let state = crypto.randomBytes(40).toString("hex");
```

Comment:
This is also duplicated in https://github.com/mozilla/blurts-server/pull/29/files#diff-4e57e667bb9833a8936a9e4a87bb539eR34

```diff
router.get("/init", function(req, res) {
  // Set a random state string in a cookie so that we can verify
  // the user when they're redirected back to us after auth.
+ let state = crypto.randomBytes(40).toString("hex");
  let uri = FxAOAuth.code.getUri({state});
  req.session.state = state;
  res.redirect(uri);
});
```

Not sure if they're related, and we should extract into some helper function, or if it's simply two completely separate places we need random strings.